viz/README.md

```diff
@@ -40,11 +23,54 @@ Copy over the `opflowout.json` file to the `viz/data` subdirectory. Next, run th
 python geninputfile.py opflowout.json
 ```
 
-You are ready to launch the visualization now. 
-
 Note: If you have created the JSON file externally then simply copy it over in the `viz/data` subdirectory and run the `geninputfile.py` script using the above command.
 
-## Launch visualization
+## Preparing the database for query
+Behind the scenes, LLM translates natural language queries into SQL queries to retrieve data from a database. As the power grid network is a typical geospatial dataset, we choose PostgreSQl + PostGIS database for the convenience of conducting spatial queries. Please follow the steps below to set up your PostgreSQL database that contains the power grid network dataset. 
+
+First, we need to convert the ExaGO output `.json` files to `.csv` files. The difference between the two data formats is that JSON stores attributes and values as dictionary pairs but CSV stores attributes and values as tables. You can write your own script for this conversion or use the provided script. 
+
+To use the provided script, first copy the ExaGO output `.json` file to the `viz/data` subdirectory and simply run the following script in the `viz/data` subdirectory (replace the example filename with your json filename). This will output three CSV files: `generation.csv`, `bus.csv`, and `tranmission_line.csv`.
+```
+python jsontocsv.py opflowout.json
+```
```

Comment:
@abhyshr iirc we discussed adding direct CSV output support to both ExaGO and the Python bindings at some point, but were satisfied with this approach (post-processing based conversion). Should we open an issue explicitly to track this?